tensorflow/core/kernels/mkl_batch_matmul_op.cc

```diff
@@ -25,7 +25,7 @@ limitations under the License.
 
 #define EIGEN_USE_THREADS
 
-#if defined(INTEL_MKL)
+#if defined(INTEL_MKL) && !defined(ENABLE_MKLDNN_V1)
```

Comment:
This will completely disable MklBatchMatMul and will cause a run time error when the runtime tires to execute the kernel for MklBatchMatMul.

Reply:
Presumably this currently the case for a TF 2.2 build with oneDNN v0.x as `!defined(INTEL_MKL_DNN_ONLY)` is included [here](https://github.com/tensorflow/tensorflow/blob/2b96f3662bd776e277f86997659e61046b56c315/tensorflow/core/kernels/mkl_batch_matmul_op.cc#L28) up until [this commit](https://github.com/tensorflow/tensorflow/commit/0616e14c5446982299407f86efc4d4751605621d#diff-3c9caa34f5e420e01358e018e4bb54c6)?